shinysdr/plugins/rtty/__init__.py

```diff
@@ -55,59 +56,105 @@
 
 
 class RTTYDemodulator(gr.hier_block2, ExportedState):
+    '''Demodulate typical amateur RTTY.
+
+    Assumptions:
+
+        - 45.45 baud
+        - 170 Hz spacing
+        - Mark tone high
+
+    TODO: make these assumptions parameters.
+    '''
+
     implements(IDemodulator)
     
-    __filter_low = 1500
-    __filter_high = 2500
-    __transition = 100
+    _spacing = 170
+
+    _low_cutoff = _spacing * -2
+    _high_cutoff = _spacing
+    _transition_width = _spacing
 
-    def __init__(self, mode,
-            input_rate=0,
-            context=None):
+    def __init__(self, mode, input_rate=0, context=None):
         assert input_rate > 0
+        self.input_rate = input_rate
         gr.hier_block2.__init__(
             self, 'RTTY demodulator',
             gr.io_signature(1, 1, gr.sizeof_gr_complex * 1),
             gr.io_signature(1, 1, gr.sizeof_float * 1),
         )
-        self.__text = u''
-        
-        baud = _DEFAULT_BAUD  # TODO param
-        self.baud = baud
 
-        demod_rate = 6000  # TODO optimize this value
-        self.samp_rate = demod_rate  # TODO rename
+        self.samp_rate = 6000
```

Comment:
`demod_rate`, or in this case `__demod_rate`, is the preferred name for a value with this role